src/openforms/prefill/api/urls.py

```diff
@@ -10,3 +10,10 @@
         name="prefill-attribute-list",
     ),
 ]
+
+urlpatterns += [
```

Comment:
can you add the comment about registering plugin-specific endpoints and the TODO that this should be made properly dynamic, same as with the registrations module?